packages/logger/src/Logger.ts

```diff
@@ -434,7 +451,7 @@ class Logger extends Utility implements LoggerInterface {
         } finally {
           Logger.injectLambdaContextAfterOrOnError(
             loggerRef,
-            initialPersistentAttributes,
+            loggerRef.getPersistentLogAttributes(),
             options
           );
```

Comment:
Let's change the call for this method to:

```ts
 Logger.injectLambdaContextAfterOrOnError(
  loggerRef,
  {},
  options
);
```

The second argument is there for backwards compatibility (we'll mark `injectLambdaContextAfterOrOnError` as deprecated as well), but it can be just an empty object so we save an operation.